portfolio/src/main/webapp/index.html

```diff
@@ -7,24 +7,54 @@
     <script src="script.js"></script>
   </head>
   <body>
+
+    <div class='parallax' id='home'>
+        <div class='dot'>
+            <p> <b> Cindy Li's <br> Portfolio </b> </p>
+        </div>
+
+        <div class='border'> </div>
+    </div>
+
+    <div class='nav'>
+        <a href='index.html#home' class='active'>Home</a>  
+        <a href='index.html#content'>About Me</a>  
+        <a href = 'projects.html'> My Projects</a> 
```

Comment:
consistent spacing for attribute assignment please, not commenting this everywhere, but please fix. Also you probably don't want the space before "My Projects"